.circleci/config.yml

```diff
@@ -144,13 +144,15 @@ jobs:
       - image: circleci/python:3.6-stretch-node-browsers
         environment:
           PYTHON_VERSION: py36
+          PERCY_ENABLE: '0'
```

Comment:
<img width="545" alt="Screen Shot 2019-08-12 at 8 38 53 PM" src="https://user-images.githubusercontent.com/2678795/62907448-37e83700-bd41-11e9-8a19-d582e05c1d62.png">

I guess also take `percy-finalize` out of the 2.7 and 3.6 workflows?